docs/docs/guides/web3_plugin_guide/plugin_users.md

```diff
@@ -9,42 +9,55 @@ This guide intends to provide the necessary context for registering plugins with
 
 ## Installing the Plugin
 
-Unless otherwise mentioned by the plugin author, installing a plugin should be as simple as `yarn add plugin-name`. This should add the plugin as a dependency within your `package.json` and the plugin should be available to import within your code.
+Unless otherwise mentioned by the plugin author, installing a plugin should be as simple as `yarn add web3-sample-plugin`. This should add the plugin as a dependency within your `package.json` and the plugin should be available to import within your code.
 
 ```json
+# package.json
 {
-	"name": "your-package-name",
-	"version": "0.0.1",
+	...
 	"dependencies": {
-		"web3-plugin": "0.1.0"
+		"web3-sample-plugin": "0.1.0"
 	}
 }
 ```
 
 ## Registering the Plugin
 
-The `.registerPlugin` method is what you're going to be using to add a plugin to an instance of a class sourced from web3.js' modules (i.e. `Web3` or `Web3Eth`). This method only exists on classes that extend `Web3Context`, so it may not be available on every class you import from a Web3.js package.
+To add a plugin to an instance of a class sourced from web3.js' modules (such as `Web3` or `Web3Eth`), you will use the `.registerPlugin` method. It's important to note that this method is only available on classes that extend `Web3Context`, so it may not be available on every class you import from a Web3.js package.
 
-The following is an example of registering the plugin `SimplePlugin` onto an instance of `Web3`:
+For illustration purposes, let's assume a plugin developer has the following code for their plugin. Please note that this code should not be touched by the plugin user:
 
 ```typescript
-import { Web3PluginBase } from 'web3-core';
+// code written by the plugin **developer**
 
-export class SimplePlugin extends Web3PluginBase {
-	public pluginNamespace = 'simplePlugin';
+import { Web3PluginBase } from 'web3';
 
-	public simpleMethod() {
+export class SamplePlugin extends Web3PluginBase {
```

Comment:
Could rename this to `PluginSample` to match `web3-plugin-sample`, but I don't it's a must

Reply:
Done in #6182